### ISSUE TITLE
Fix package import logging.config

### ISSUE THREAD

**Issue by jgergo** [tintoy/seqlog#10]
* PySeq version: 0.3.9
* Python version: 3.7.0
* Operating System: Win10

### Description
I tried to use the .yaml based configuration of the logger, with the same config file as provided in the documentation.
The seqlog.configure_from_file method fails with an AttributeError: module 'logging' has no attribute 'config'.
Based on the docs, the logger should be configured after the command.

### What I Did
```
file_name = r"\config.logger.yaml"
config_file_location = working_dir + file_name
seqlog.configure_from_file(config_file_location)
```

```
Traceback (most recent call last):
  File "C:/_repos/aadpackages/AAD_ExcelChecker/central_model_logging/seq_config_from_yaml.py", line 11, in <module>
    seqlog.configure_from_file(config_file_location)
  File "C:\_repos\aadpackages\AAD_ExcelChecker\venv\lib\site-packages\seqlog\__init__.py", line 33, in configure_from_file
    configure_from_dict(config, override_root_logger)
  File "C:\_repos\aadpackages\AAD_ExcelChecker\venv\lib\site-packages\seqlog\__init__.py", line 51, in configure_from_dict
    logging.config.dictConfig(config)
AttributeError: module 'logging' has no attribute 'config'
```
### Proposed fix
modify the package.__init__ file by adding:

`import logging.config `
 
to the package imports

### Secondary issue
after using the proposed fix, only the root logger can take keyword arguments in the logging.

```
seqlog.configure_from_file(config_file_location)
root_logger = logging.getLogger()
root_logger.info("this is the {logger}", logger="root")

another_logger = logging.getLogger("another_logger")
another_logger.info("This is {logger} logger.", logger="root")
```

```
Traceback (most recent call last):
  File "C:/_repos/aadpackages/AAD_ExcelChecker/central_model_logging/seq_config_from_yaml.py", line 14, in <module>
    another_logger.info("This is {logger} logger.", logger="root")
  File "C:\Users\gjedlicska\AppData\Local\Programs\Python\Python37-32\lib\logging\__init__.py", line 1332, in info
    self._log(INFO, msg, args, **kwargs)
TypeError: _log() got an unexpected keyword argument 'logger'
```

Sending a message without keyword arguments works as expected.
`another_logger.info("This is another_logger logger.")`

**Comment by tintoy** [comment]
Oops! Sorry about that; I'll fix it and publish a new version shortly.

As for the second issue, that's unexpected (and quite interesting) - would you mind checking whether keyword arguments work for you when [programmatically configuring](https://seqlog.readthedocs.io/en/latest/usage.html#configure-logging-programmatically) the logger?

**Comment by tintoy** [comment]
Ok, pretty sure I've fixed both issues. Would you mind taking a look at PR #11 to see if it makes sense to you?

**Comment by tintoy** [comment]
Published [v0.3.10](https://pypi.org/project/seqlog/0.3.10/) :smile:

**Comment by tintoy** [comment]
As noted in the PR, the `class` annotation is not respected by Python's `logging.config` on individual `logger` entries (although it _is_ respected on `handler` entries). I've therefore removed those `class` annotations from the example config in the docs.
